homeassistant/components/blink/__init__.py

```diff
@@ -84,6 +85,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     auth_data = deepcopy(dict(entry.data))
     blink.auth = Auth(auth_data, no_prompt=True, session=session)
     blink.refresh_rate = entry.options.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
+    coordinator = BlinkUpdateCoordinator(hass, blink)
+    hass.data[DOMAIN][entry.entry_id] = coordinator
```

Comment:
Oh I now notice this thing, please first call the `async_config_entry_first_refresh` and then assign the coordinator to hass.data. Otherwise you could be storing a non-functional coordinator in the hass.data

Reply:
makes sense, but it doesn't appear to change how it operates from what I can tell.

Reply:
It doesn't but its just a best practice. Otherwise you're storing an object that doesn't function in memory. If more integrations would do this you might notice it at some point.